velox/common/memory/tests/MemoryPoolTest.cpp

```diff
@@ -71,6 +72,7 @@ class MemoryPoolTest : public testing::TestWithParam<TestParam> {
  protected:
   static void SetUpTestCase() {
     FLAGS_velox_memory_leak_check_enabled = true;
+    FLAGS_velox_memory_debug_mode_enabled = true;
     TestValue::enable();
   }
```

Comment:
Shall we have a dedicated test case to check if the call sites have been allocated as expected: (1) the number of call sites and (2) call site is not empty? You can provide a testingMethod to MemoryPoolImpl for this purpose.

Reply:
Yeah we can do that.